using-mobile-extensions/places-extension/places-monitoring-extension/using-your-own-monitor.md

```diff
@@ -1,4 +1,142 @@
 # Using your own monitor
 
-This information is  in progress.
+You can also use your own monitoring services and intergrate with Places using the Places Extension API's
 
+## Registering Geofences
+
+In the occasion of using your own monitoring services, register the geofences of the near by point of intrests around the current location by implementing the following steps.
+
+{% tabs %}
+
+{% tab title="iOS" %}
+
+1. The location updates obtained from the Core Location services of the iOS should be passed to the Places Extension. Use the `getNearbyPointsOfInterest` Places Extension API to get the array of n  `ACPPlacesPoi` objects around the current location.
```

Comment:
Please change step 1 so that it reads as follows:

"1. Pass the location updates that were obtained from the Core Location services of the iOS to the Places Extension. 
2. Use the `getNearbyPointsOfInterest` Places Extension API to get the array of n  `ACPPlacesPoi` objects around the current location."